backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/LabelComponent.java

```diff
@@ -71,9 +73,9 @@ public Element render() {
         LabelElementProps labelElementProps = LabelElementProps.newLabelElementProps(id)
                 .type("label:inside-center") //$NON-NLS-1$
                 .text(text)
-                .position(Position.UNDEFINED)
-                .size(Size.UNDEFINED)
-                .alignment(Position.UNDEFINED)
+                .position(optionalPreviousLabel.map(Label::getPosition).orElse(labelBoundsProvider.getPosition()))
+                .size(optionalPreviousLabel.map(Label::getSize).orElse(labelBoundsProvider.getSize()))
+                .alignment(optionalPreviousLabel.map(Label::getAlignment).orElse(labelBoundsProvider.getAlignment()))
```

Comment:
Extract that in variable, it's not very readable

Reply:
see 0d825fc825323b2902b76c564e851eec89a13e82